files/home/mastodon/live/lib/tasks/digital_ocean.rake

```diff
@@ -40,7 +40,40 @@ namespace :digitalocean do
       env['REDIS_PORT'] = 6379
 
       if prompt.yes?('Do you want to store user-uploaded files on the cloud?', default: false)
-        case prompt.select('Provider', ['Amazon S3', 'Wasabi', 'Minio', 'Google Cloud Storage'])
+        case prompt.select('Provider', ['DigitalOcean Space','Amazon S3', 'Wasabi', 'Minio', 'Google Cloud Storage'])
+        when 'DigitalOcean Space'
```

Comment:
And here

Reply:
I am pretty sure it is okay to refer to it as singular "Space" below.

Reply:
Are you sure people will recognise it? I think just 'Space' is not that widely used in community

Reply:
I checked with Amanda and using singular "Space"is okay here, but the product name is "DigitalOcean Spaces"

Reply:
forwarding for anyone reading this: We ended up using `DigitalOcean Spaces` as name of prompt entry. And just `Space` for connection attributes